shared/stories/stories.js

```diff
@@ -1,76 +1,28 @@
 // @flow
-import avatar from '../common-adapters/avatar.stories'
-import meta from '../common-adapters/meta.stories'
-import buttonBar from '../common-adapters/button-bar.stories'
-import button from '../common-adapters/button.stories'
-import box from '../common-adapters/box.stories'
-import chatSendAnimation from '../chat/conversation/messages/wrapper/chat-send.stories'
-import chatInboxRow from '../chat/inbox/row/index.stories'
-import chatInfoPanel from '../chat/conversation/info-panel/index.stories'
-import chatInfoPanelNotifications from '../chat/conversation/info-panel/notifications/index.stories'
-// import chatList from '../chat/conversation/list/index.stories'
-import chatCreateChannel from '../chat/create-channel/index.stories'
-import chatManageChannels from '../chat/manage-channels/index.stories'
-import chatChannelMentionHud from '../chat/conversation/input-area/channel-mention-hud/index.stories'
-import chatUserMentionHud from '../chat/conversation/input-area/user-mention-hud/index.stories'
-import checkbox from '../common-adapters/checkbox.stories'
+import chat from '../chat/index.stories'
```

Comment:
Looks like we have both 'Chat/Channel Heads up display' and 'Chat/Heads up display' -- merge them?

Reply:
Never mind!